components/phpunit_bridge.rst

```diff
@@ -165,6 +165,9 @@ Deprecation notices can be triggered by using::
 
     @trigger_error('Your deprecation message', E_USER_DEPRECATED);
 
+You can also require the `symfony/deprecation-contracts` that provide the global
+function `trigger_deprecation()` for this usage.
```

Comment:
```suggestion
``trigger_deprecation()`` function for this usage.
```

I would rather use:
```rst
_method:`\Symfony\........::trigger_deprecation`
```

here

Reply:
let me find an example in the codebase @OskarStark :)

Reply:
hum in fact there is no namespace etc, it is a global func, so what path should i write?

Reply:
Lets keep it like this then, thanks!